packages/gguf/src/quant_descriptions.ts

```diff
@@ -0,0 +1,26 @@
+import { GGMLQuantizationType } from "./types";
+
+export const QUANT_DESCRIPTIONS: Record<GGMLQuantizationType, string> = {
+	[GGMLQuantizationType.F32]: "32-bit standard IEEE 754 single-precision floating-point number.", // src: https://en.wikipedia.org/wiki/Single-precision_floating-point_format
+	[GGMLQuantizationType.F16]: "16-bit standard IEEE 754 half-precision floating-point number.", // src: https://en.wikipedia.org/wiki/Half-precision_floating-point_format
+	[GGMLQuantizationType.Q4_0]: "", // todo: add description
+	[GGMLQuantizationType.Q4_1]: "", // todo: add description
+	[GGMLQuantizationType.Q5_0]: "", // todo: add description
+	[GGMLQuantizationType.Q5_1]: "", // todo: add description
+	[GGMLQuantizationType.Q8_0]: "", // todo: add description
+	[GGMLQuantizationType.Q8_1]: "", // todo: add description
+	[GGMLQuantizationType.Q2_K]: `"type-1" 2-bit quantization in super-blocks containing 16 blocks, each block having 16 weight. Block scales and mins are quantized with 4 bits. This ends up effectively using 2.5625 bits per weight (bpw). In "type-1", weights are given by w = d * q + m, where m is the block minimum.`, // src: https://github.com/ggerganov/llama.cpp/pull/1684#issue-1739619305
```

Comment:
should you encode the `src` link in the code itself (so a `Record<GGMLQuantizationType, { txt: string; url: string }>` to be able to link to a reference from the UI?

Reply:
Another potential idea: indicate a few of them with a `featured` or `popular` flag so we can showcase in a UI or something

Reply:
handled in https://github.com/huggingface/huggingface.js/pull/615/commits/240f0df6f2f6ab03c577dad21e733360e0aabdf9